pkgs/development/python-modules/pyment/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "Pyment";
+  version = "0.3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "951a4c52d6791ccec55bc739811169eed69917d3874f5fe722866623a697f39d";
+  };
+
+  doCheck = false;
```

Comment:
That's because the tests directory is not included in the PyPI tarball. Please include an appropriate comment.
Also see https://github.com/dadadel/pyment/pull/75.